src/views/partials/signout-bar.njk

```diff
@@ -4,5 +4,5 @@
         <span class="content-email" id="signed-in-user">{{email}}</span>
         <a class="govuk-link" href="{{ExternalUrls.SIGNOUT}}" data-event-id="sign-out-button-selected" id="user-signout">Sign Out</a>
     </div>
-<script src="//{{ cdn.host }}/javascripts/app/session-timeout.js"></script>
+<script src="{{ cdn.host }}/javascripts/app/session-timeout.js"></script>
```

Comment:
The protocol relative `//` are not the problem. 
The problem is that the variable `cdn.host` is not defined so it is being replaced with an empty string. 
This needs to be replaced with the correct variable `cdnHost`.
It also needs to be checked if the cdnHost already has a protocol. If so the `//` needs to be removed as welll.